Detectors/ITSMFT/ITS/macros/test/run_buildTopoDict_its.C

```diff
@@ -202,12 +202,8 @@ void run_buildTopoDict_its(std::string clusfile = "o2clus_its.root",
   TCanvas* cComplete = new TCanvas("cComplete", "Distribution of all the topologies");
   cComplete->cd();
   cComplete->SetLogy();
-  TH1F* hComplete = (TH1F*)dict.mHdist.Clone("hDict");
-  hComplete->SetDirectory(0);
-  hComplete->SetTitle("Topology distribution");
-  hComplete->GetXaxis()->SetTitle("Topology ID");
-  hComplete->SetFillColor(kRed);
-  hComplete->SetFillStyle(3005);
+  TH1F* hComplete = nullptr;
+  o2::itsmft::TopologyDictionary::getTopologyDistribution(dict.getDictionary(), hComplete, "hComplete");
   hComplete->Draw("hist");
```

Comment:
The histo is not shown in the canvas after the macro is run (but is visible in the pdf). The reason is that you create it in the file and it is deleted from memory and canvas when the file is closed. Please add right after its creation:
``hComplete->SetDirctory(0);``.
The same is for other places with similar lines.

Reply:
Yes, I alway run in batch mode and I did not realise